dev/com.ibm.ws.microprofile.health/resources/com/ibm/ws/microprofile/health/resources/Health.nlsprops

```diff
@@ -47,7 +47,7 @@ healthcheck.bean.call.exception.CWMH0050E.explanation=The specified call method
 healthcheck.bean.call.exception.CWMH0050E.useraction=Review your call method within the HealthCheck bean to identify the exception that occurred.
 
 # The HealthCheck bean call method had a DOWN outcome
-healthcheck.application.down.CWMH0052W=CWMH0052W: The {0} implementing HealthCheckResponse in the {1} application in module {2}, reported a {3} status with data {4}.
+healthcheck.application.down.CWMH0052W=CWMH0052W: The {0} implementing HealthCheckResponse in the {1} application in module {2}, reported a {3} status for the health check with name {4} and with data {5}.
```

Comment:
The {0} HealthCheckResponse in the {1} application in the module, {2}, reported a {3} status for the health check, {4}, and with the data {5}.

Reply:
Fixed.